### PR TITLE
feat: add vehicle info util

### PR DESCRIPTION
**Pull request by taikitanaka3**
# Related Issue

#1

# Description
Add following tier4 pkg from https://github.com/tier4/AutowareArchitectureProposal.iv

vehicle_info_util

### CODE REVIEW

File: vehicle/vehicle_info_util/launch/vehicle_info.launch.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    vehicle_param_file = LaunchConfiguration("vehicle_info_param_file").perform(context)
+    with open(vehicle_param_file, "r") as f:
+        vehicle_param = yaml.safe_load(f)["/**"]["ros__parameters"]
+    return [SetParameter(name=k, value=v) for (k, v) in vehicle_param.items()]
+
+
+def generate_launch_description():
+    return LaunchDescription(
+        [
+            DeclareLaunchArgument("vehicle_info_param_file"),
```
Comment (tkimura4) [inline vehicle/vehicle_info_util/launch/vehicle_info.launch.py:33]: Could you create default vehicle_info_param_file and set it as default parameter?

Comment (taikitanaka3) [reply vehicle/vehicle_info_util/launch/vehicle_info.launch.py:33]: added lexus description as default
c096097

Comment (tkimura4) [reply vehicle/vehicle_info_util/launch/vehicle_info.launch.py:33]: Thanks!
Then, Could you set it as a default parameter in vehicle_info.launch.py?

Comment (taikitanaka3) [reply vehicle/vehicle_info_util/launch/vehicle_info.launch.py:33]: @tkimura4 
I added sample launch file at 58688eb

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**